readhub/src/main/java/com/fmiunibuc/readhub/model/Book.java

```diff
@@ -0,0 +1,26 @@
+package com.fmiunibuc.readhub.model;
+
+import com.sun.istack.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+public class Book {
+    @Id
+    @GeneratedValue
+    private Long id;
+    @NotNull
+    private String name;
+    private String author;
+
```

Comment:
we should add status field to Book class

Reply:
I think it's better to keep the classes as simple as possible for now. I'll make sure to add the status field in the todo list and we can resolve this before connecting the database.

Reply:
ok